versions.tf

```diff
@@ -20,7 +20,17 @@ terraform {
 
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.53, < 5.0"
+      version = ">= 3.53, < 6"
+    }
+
+    local = {
+      source  = "hashicorp/local"
+      version = "~> 2.4"
```

Comment:
Could be considered a breaking change? Came out after the most recent release of this module.